test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py

```diff
@@ -122,3 +126,20 @@ def step(context, message_index):
 def step(context, amount):
     _statusCommunityScreen.check_pin_count(amount)
 
+@Then("the user blocks member with name |any|")
+def step(context, member):
```

Comment:
Same thing for the others. It makes it clearer that we refer to the name
```suggestion
def step(context, member_name):
```